lesson13/main.js

```diff
@@ -1,22 +1,27 @@
 const ul = document.getElementById("js-ul");
-const button = document.getElementById("js-button");
+const fetchButton = document.getElementById("js-fetch-button");
+const errorMessage = document.createElement("p");
+const openButton = document.getElementById("js-open-button");
+const closeButton = document.getElementById("js-close-button");
+const backButton = document.getElementById("js-back-button");
+const modal = document.getElementById("js-modal");
+const mask = document.getElementById("js-mask");
+
 const url = "https://mocki.io/v1/1c058349-634e-462a-ad37-14f135e59b99";
 // const url = ""; //Not JSON
 // const url = "https://mocki.io/v1/55dc6233-a8fe-44ca-8906-3de313545ce8"; //No data
 // const url = "https://mocki.io/v1/1c058349-634e-"; //Failed to fetch
 
 function renderStatus(response) {
-  const p = document.createElement("p");
-  p.id = "render-status";
-  p.textContent = `${response.status}:${response.statusText}`;
+  errorMessage.id = "render-status";
+  errorMessage.textContent = `${response.status}:${response.statusText}`;
   document.body.appendChild(p);

```

Comment:
MUST
p does not exist.
```suggestion
  document.body.appendChild(errorMessage);

```

Reply:
Thank you mai-san :）
I fixed that.
8c01f03f1e79dbc00a9abd3826691c8b67d8c28a